Meniga/Meniga-ios-sdk/Internal/Utilities/Extensions/NSDateUtils.m

```diff
@@ -34,13 +34,10 @@ +(BOOL)isDate:(NSDate*)firstDate equalToDate:(NSDate *)secondDate withDateCompon
 
 +(BOOL)isDate:(NSDate*)firstDate equalToDayMonthAndYear:(NSDate *)secondDate {
     
-    NSDateComponents *dateComponents = [[NSCalendar currentCalendar] components: NSCalendarUnitMonth | NSCalendarUnitYear | NSCalendarUnitDay fromDate:firstDate toDate:secondDate options:0 ];
+    NSDateComponents *firstDateComponents = [[NSCalendar currentCalendar] components:NSCalendarUnitYear | NSCalendarUnitMonth | NSCalendarUnitDay fromDate:firstDate];
```

Comment:
Minor: you can extract unit options to variable and use it for first and second components:

```
NSCalendarUnit calendarUnits = NSCalendarUnitYear | NSCalendarUnitMonth | NSCalendarUnitDay;
NSDateComponents *firstDateComponents = [[NSCalendar currentCalendar] components: calendarUnits fromDate:firstDate];
NSDateComponents *secondDateComponents = [[NSCalendar currentCalendar] components: calendarUnits fromDate:secondDate];
```